Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/helper.go

```diff
@@ -320,6 +320,11 @@ func NewOrDie(c *Config) *Client {
 // running inside a pod running on kuberenetes. It will return an error if
 // called from a process not running in a kubernetes environment.
 func InClusterConfig() (*Config, error) {
+	host, port := os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")
```

Comment:
That's definitely something we can go do in the future.  It's just not valid unless we do this today.